### DISCUSSION TITLE
how can i add the French blocklist from AdGuard Filters from Github?

### DISCUSSION THREAD

**Discussion by raykai** [AdguardTeam/AdGuardHome#3573] [Q&A] [answered]
Hi, How can i add the French block list found at:
[https://github.com/AdguardTeam/AdguardFilters/tree/master/FrenchFilter/sections](https://github.com/AdguardTeam/AdguardFilters/tree/master/FrenchFilter/sections) to AdGard Home?

It would be nice to have more regional blocklists in AdGard Home seeing as there are many lists here: https://github.com/AdguardTeam/AdguardFilters I do not know why they are not all in AdGard Home blocklists.

**Comment by ameshkov** [comment, marked as answer]
Note that the French filter there is not supposed to be used in a DNS-level blocker.

However, the ad domains from it are used in AdGuard DNS filter: https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/configuration.json

**Reply by raykai** [reply]
so its on by default?

**Reply by ameshkov** [reply]
Yep
